wait_for_it/wait_for_it.py

```diff
@@ -98,6 +91,13 @@ async def _wait_until_available_and_report(reporter, host, port):
     show_default=True,
     help="Timeout in seconds, 0 for no timeout",
 )
+@click.option(
+    "-s",
+    "--service",
+    metavar="host:port",
+    multiple=True,
+    help="Services to test, in one of the formats 'host:port', '[v6addr]:port' or 'https://...'",
```

Comment:
```suggestion
    help="Services to test, in one of the formats 'hostname:port', 'ipv4:port', '[ipv6]:port' or 'http://...'",
```

Reply:
Ignore the regex suggestion. It is probably too complicated. Just add in the ipv4 example and it should be clear.

Reply:
I have applied `s/host/hostname/` and added IPv4 now, also rebased. Let's no advertise unprotected TLS in 2020 so I kept "https" with "s". If you insist, okay, but I think "https" is the better choice in 2020.

Reply:
sgtm

Reply:
> Let's no advertise unprotected TLS in 2020 so I kept "https" with "s". If you insist, okay, but I think "https" is the better choice in 2020.

There is a need to connect to both `http` and `https` services. Supporting `http` connection in itself is not a security issue. To the tool, the only thing that matters is that a network connection can be established. If the service owner  wants to restrict to only TLS connections that is on them.

Reply:
I agree. The marginal benefit is that we help burn https into peoples minds. We had to make a choice between http and https.

Reply:
Got it, thanks for clarifying! For the reasons, you mention `https` in the text does make more sense. My intention with the comment was to gauge whether or not there was a serious security issue on our hands and if we need a GH issue created to track. Thanks @hartwork !